test/testSamples/testStandardSurface/testStandardSurface.ma

```diff
@@ -112,9 +111,6 @@ createNode file -n "file1";
 	setAttr ".cs" -type "string" "sRGB";
 createNode place2dTexture -n "place2dTexture1";
 	rename -uid "6616C90D-4B36-08C7-8666-33BB13D15663";
-createNode mayaUsdLayerManager -n "mayaUsdLayerManager1";
-	rename -uid "61CE7C31-4E04-09B7-BFEA-F291A58D7E4C";
-	setAttr ".sst" -type "string" "";
```

Comment:
Are these changes required? Not sure why they're here, doesn't look like they're used

Reply:
This's just for clearing out mayausd stuff, this might cause loading of mayausd, which is not necessary for this case.